doc/user/content/ingest-data/postgres-alloydb.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Ingest data from AlloyDB"
+description: "How to stream data from AlloyDB to Materialize"
+menu:
+  main:
+    parent: "postgresql"
+    name: "AlloyDB for PostgreSQL"
```

Comment:
Since the parent is `Postgres` I think is ok to remove the latter.
```suggestion
    name: "AlloyDB"
```

Reply:
I think that 'AlloyDB for PostgreSQL' is the official name of the product:

<img width="283" alt="image" src="https://github.com/MaterializeInc/materialize/assets/21223421/17814dc9-c634-485e-886c-17c9c1038647">

Though open to suggestions if we should shorten the official name and only use AlloyDB?

Reply:
AlloyDB is the engine as CosmosDB is the engine for Azure, or Aurora ([Aurora for Postgres in our docs](https://materialize.com/docs/ingest-data/postgres-amazon-aurora/)) for AWS. PostgresSQL is the protocol. I presume they display only the 'AlloyDB for PostgreSQL' because they haven't released other protocols yet like Azure or AWS have.